### PR TITLE
Update tsp-cli-client, README, requirements &co based on current Code IDE use

### PR DESCRIPTION
**Pull request by marco-miller**

### CODE REVIEW

**Review by MatthewKhouzam — APPROVED** [review]
[approved] README: Fix the markdownlint problems from VS Code

**Review by MatthewKhouzam — APPROVED** [review]
[approved]  Add VS Code python virtual-env folder to gitignore

**Review by MatthewKhouzam — APPROVED** [review]
[approved] README: Fix the markdownlint problems from VS Code

**Review by MatthewKhouzam — APPROVED** [review]
[approved]  README: Remove blank lines at end of python block

**Review by MatthewKhouzam — APPROVED** [review]
[approved] Sort required modules alphabetically to ease reads

File: tsp-cli-client [hunk 1]
```diff
@@ -64,7 +64,7 @@ def print_time(elem):
         if ns > 1000:
             return '{0} µs'.format(ns / 1000)
         return '{0} ns'.format(ns)
-    except ValueError as e:
+    except ValueError:
         return elem
```
Comment (MatthewKhouzam) [inline tsp-cli-client:72]: can we log this instead?

**Review by MatthewKhouzam — CHANGES_REQUESTED** [review]
Small nit

**Review by MatthewKhouzam — COMMENTED** [review]
[approved]  tsp-cli-client: Base python3 binary on local env's

**Review by MatthewKhouzam — APPROVED** [review]
[approved]  Fix panda(s) module name typo in requirements.txt

**Review by MatthewKhouzam — APPROVED** [review]
[approved]  README: Match current './tsp-cli-client -h' output

**Review by MatthewKhouzam — APPROVED** [review]
[approved] tsp-cli-client: Fix minor verb typo in description

**Review by MatthewKhouzam — APPROVED** [review]
[approved]  tsp-cli-client: Format w/ default VS Code autopep8 

Looks clean to me.

**Review by MatthewKhouzam — CHANGES_REQUESTED** [review]
Github ate my "request changes"

File: tsp-cli-client (hunk 1 above)
Comment (marco-miller) [reply tsp-cli-client:72]: > can we log this instead?

Good suggestion; thanks.
This should be another commit though, as this very one focuses on fixing lint issues solely.
Indeed, 'e' is not used, hence the lint issue fixed herein.

Comment (marco-miller) [reply tsp-cli-client:73]: -Done; I amended this specific commit thus rebased its follow-ups.
(I also pushed some more small changes on top.)

Comment (marco-miller) [reply tsp-cli-client:73]: <=> https://github.com/theia-ide/tsp-python-client/compare/4e4de950c06cc6c9cc0096ae95702b1868188226..9667859e8247c4c1a8860a5f7d07cfac238e0005

**Review by MatthewKhouzam — APPROVED** [review]

**Review by MatthewKhouzam — APPROVED** [review]

**Review by MatthewKhouzam — APPROVED** [review]

**Review by MatthewKhouzam — APPROVED** [review]

**Review by MatthewKhouzam — APPROVED** [review]

**Comment by MatthewKhouzam** [comment]
These are trivial no-brainers. I will merge them.

**Pull request merged**